cdc/model/sink.go

```diff
@@ -466,36 +466,13 @@ func ColumnValueString(c interface{}) string {
 	return data
 }
 
-// ColumnInfo represents the name and type information passed to the sink
-type ColumnInfo struct {
-	Name string `msg:"name"`
-	Type byte   `msg:"type"`
-}
-
-// FromTiColumnInfo populates cdc's ColumnInfo from TiDB's model.ColumnInfo
-func (c *ColumnInfo) FromTiColumnInfo(tiColumnInfo *model.ColumnInfo) {
-	c.Type = tiColumnInfo.GetType()
-	c.Name = tiColumnInfo.Name.O
-}
-
-// SimpleTableInfo is the simplified table info passed to the sink
-type SimpleTableInfo struct {
-	// db name
-	Schema string `msg:"schema"`
-	// table name
-	Table string `msg:"table"`
-	// table ID
-	TableID    int64         `msg:"table-id"`
-	ColumnInfo []*ColumnInfo `msg:"column-info"`
-}
-
 // DDLEvent stores DDL event
 type DDLEvent struct {
 	StartTs      uint64           `msg:"start-ts"`
 	CommitTs     uint64           `msg:"commit-ts"`
-	TableInfo    *SimpleTableInfo `msg:"table-info"`
```

Comment:
Will this cause a compatibility issue? Because we use it in `msg`

Reply:
There are two solutions to this compatibility issue:
- The redo-enabled changefeed should be  pause & resumed, therefore the existing redo DDL & row events can be cleared.
- Wait 5 seconds until the existing redo DDL & row events can be cleared.

Reply:
This compatibility issue only occurs when applying redo log right after a TiCDC upgrade (< 5s). I think it's relatively rare.